uber/config.py

```diff
@@ -344,4 +357,4 @@ def _is_intstr(s):
 
 c.SAME_NUMBER_REPEATED = r'^(\d)\1+$'
 
-stripe.api_key = c.STRIPE_SECRET_KEY
+stripe.api_key = c.STRIPE_SECRET_KEY
```

Comment:
Actually PEP8 requires a newline at the end of the file anyway, which is why Travis is failing :)